scripts/cibuild

```diff
@@ -25,19 +23,19 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     else
-        ./scripts/test
+        docker-compose up -d database
 
-        pushd "${DIR}"
+        ./scripts/test
 
         echo "Building JAR for API subproject"
-        ./sbt api/assembly
-
-        popd
+        docker-compose \
+            run --rm sbt \
+            "api/assembly"
 
         echo "Building Scala container images"
         GIT_COMMIT="${GIT_COMMIT}" docker-compose \
             -f docker-compose.yml \
             -f docker-compose.ci.yml \
-            build api api-migrations calculate-water
```

Comment:
The `calculate-water` container image is failing to build because of a Python dependency. To allow builds to pass for the sake of this PR, I commented it out.

Reply:
It is possible that this has already been resolved upstream, but if not, I think we need to create an issue for it. I was going to test it with Jenkins, but `develop` doesn't exist in the Jenkins UI yet.

Reply:
https://github.com/raster-foundry/granary/pull/41 updated some of the `calculate-water` code, but not dependencies. I just ran `docker-compose build calculate-water` locally and it built fine so I think something else might be going on.